server/routes/bookAVisit.test.ts

```diff
@@ -2222,6 +2222,7 @@ describe('/book-a-visit/check-your-booking', () => {
           expect($('.test-additional-support').text()).toContain('Portable induction loop for people with hearing aids')
           expect($('.test-main-contact-name').text()).toContain('abc')
           expect($('.test-main-contact-number').text()).toContain('0123 456 7890')
+          expect($('form').prop('action')).toBe('/book-a-visit/check-your-booking')
```

Comment:
thinking of adding similar check to other tests to confirm form buttons posting to correct URL for book/update